packages/components/package.json

```diff
@@ -77,8 +77,8 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react": ">=16.8",
+    "react-dom": ">=16.8"
```

Comment:
we can still support both until we update our 3rd party package to latest for the ts4 upgrade